prymer/primer3/primer3_weights.py

```diff
@@ -30,23 +33,23 @@
 
 
 @dataclass(frozen=True, init=True, slots=True)
-class Primer3Weights:
+class PrimerAndAmpliconWeights:
     """Holds the weights that Primer3 uses to adjust penalties
      that originate from the designed primer(s).
 
     The weights that Primer3 uses when a parameter is less than optimal are labeled with "_lt".
     "_gt" weights are penalties applied when a parameter is greater than optimal.
 
+    Some of these settings depart from the default settings enumerated in the Primer3 manual.
```

Comment:
```suggestion
    Most of these settings depart from the default settings enumerated in the Primer3 manual.
```

**suggestion** Open an issue, to enumerate the differences (and possibly the rationale?)